planning/behavior_velocity_planner/include/utilization/util.hpp

```diff
@@ -104,10 +102,27 @@ using tier4_planning_msgs::msg::StopReason;
 
 namespace planning_utils
 {
+template <class T>
+size_t calcPointIndexFromSegmentIndex(
```

Comment:
@takayuki5168 
I think this needs size check or add only prev and next point instead of giving full path points

Reply:
We do not need to check the size because...
When the point size is 10, the max segment index is 8. So seg_idx + 1 is always smaller than 10

Reply:
@taikitanaka3

Reply:
@takayuki5168 
I understand so is there any definition for segment index which is previous index for point?